HousingFinanceInterimApi/V1/Gateways/AssetGateway.cs

```diff
@@ -16,7 +18,40 @@ public AssetGateway(DatabaseContext context)
 
         public async Task UpdateAssetDetails(UpdateAssetDetailsQuery query, UpdateAssetDetailsRequest request)
         {
-            await _context.UpdateAssetDetails(query, request).ConfigureAwait(false);
+            await UpdateAssetDetailsTable(query, request).ConfigureAwait(false);
+        }
```

Comment:
Are there any plans to extend this wrapper beyond only calling another method with the same parameters?

Reply:
None at the moment